app/controllers/posts_controller.rb

```diff
@@ -5,6 +5,31 @@ def index
   end
 
   def show
+    @like_state = false
     @post = Post.find(params[:id])
+    @post.likes.each do |like|
+      @like_state = like.author_id == current_user.id
+    end
+  end
+
+  def new
+    new_post = Post.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: new_post } }
+    end
+  end
+
+  def create
+    @new_post = Post.new(params.require(:post).permit(:text, :title))
```

Comment:
### Optional
- [ ] It would be nice if we had created the `strong params` method to handle inputs. Here is a [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) where we can read more about it and improve our existing knowledge